supervisor/daemon/myst_darwin.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package daemon
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+)
+
+const pidFile = "/var/run/myst.pid"
+
+// RunMyst runs mysterium node daemon. Blocks.
+func (d *Daemon) RunMyst() error {
+	paths := []string{
+		d.cfg.OpenVPNPath,
+		"/usr/bin",
+		"/bin",
+		"/usr/sbin",
+		"/sbin",
+		"/usr/local/bin",
+	}
+	var stdout, stderr bytes.Buffer
+	cmd := exec.Cmd{
+		Path: d.cfg.MystPath,
+		Args: []string{
+			d.cfg.MystPath,
+			"--openvpn.binary", d.cfg.OpenVPNPath,
+			"--mymysterium.enabled=false",
+			"--ui.enable=false",
+			"daemon",
+		},
+		Env: []string{
+			"HOME=" + d.cfg.MystHome,
+			"PATH=" + strings.Join(paths, ":"),
+		},
+		Stdout: &stdout,
+		Stderr: &stderr,
+	}
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+
+	err := runWithSuccessTimeout(cmd.Wait, 5*time.Second)
+	if err != nil {
+		log.Printf("myst output [err=%s]:\n%s\n%s\n", err, stderr.String(), stdout.String())
+		return err
+	}
+
+	pid := cmd.Process.Pid
+	if err := ioutil.WriteFile(pidFile, []byte(strconv.Itoa(pid)), 0700); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func runWithSuccessTimeout(f func() error, timeout time.Duration) error {
+	done := make(chan error)
```

Comment:
If there is no receiver after timeout it will be blocked forever. 
```suggestion
	done := make(chan error, 1)
```